### PR TITLE
Reviews import

### PR DESCRIPTION
**Pull request by danielSanchezQ**
Simple command that takes the vCA aggregated file and exports a csv/json file with the data ready for being imported into the VitSS DB.

### CODE REVIEW

**Comment by danielSanchezQ** [comment]
Supersedes #39

File: src/vca_reviews/mod.rs [hunk 1]
```diff
@@ -4,21 +4,14 @@ pub use tags::TagsMap;
...
 pub enum Error {
     #[error(transparent)]
-    CouldNotReadExcel(#[from] calamine::Error),
-
-    #[error(transparent)]
-    Xls(#[from] calamine::XlsxError),
-
-    #[error(transparent)]
-    CouldNotDeserialize(#[from] calamine::DeError),
+    CouldNotReadExcel(#[from] csv::Error),
```
Comment (Zeegomo) [inline src/vca_reviews/mod.rs:14]: Well, it's not and excel file anymore :)

File: src/vca_reviews/mod.rs [hunk 2]
```diff
@@ -4,21 +4,14 @@ pub use tags::TagsMap;
...
 
     #[error("Couldn't find workbook {0}")]
     CouldNotFindWorkbook(String),
```
Comment (Zeegomo) [inline src/vca_reviews/mod.rs:17]: I think this is not needed anymore

File: src/vca_reviews/mod.rs (hunk 1 above)
Comment (danielSanchezQ) [reply src/vca_reviews/mod.rs:14]: indeed

**Review by Zeegomo — APPROVED** [review]

**Pull request merged**